airbyte-integrations/connectors/source-brightpearl/source_brightpearl/schemas/product_type.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "integer"]
+    },
+    "name": {
+      "type": "string"
+    },
+    "associations": {
+      "type": ["object", "null"],
+      "properties": {
+        "options": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": ["null", "integer"]
+              },
+              "name": {
+                "type": "string"
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
Totally a nit pick, but a few of these json files could have a blank line at the end of the file.